code/modules/capture_the_flag/ctf_game.dm

```diff
@@ -40,7 +40,6 @@
 	if(!reset)
 		reset = new reset_path(get_turf(src))
 		reset.flag = src
-	RegisterSignal(src, COMSIG_PARENT_PREQDELETED, PROC_REF(reset_flag)) //just in case CTF has some map hazards (read: chasms).
```

Comment:
So what happens if they fall into a chasm now?

Reply:
> So what happens if they fall into a chasm now?

Chasm's don't qdel things anymore, so before and after this PR they just sit in their for 20 seconds then respawn, as if they were dropped by a player carrying them.